index.js

```diff
@@ -15,12 +15,11 @@ function normalizePrefix (prefixName) {
 
 class PrometheusMetrics {
   constructor (circuits, registry) {
-    if (!circuits) {
-      throw new Error('A circuit or a list of circuits is required');
+    if (circuits instanceof client.Registry) {
+      circuits = undefined;
+      registry = circuits;
```

Comment:
if you pass in a custom registry as the first parameter,  it will always be set to undefined, and then use the default `client.register`,  i'm not sure that is what you were going for here?

Reply:
I think the idea is that, you should be able to pass in just a `client.Registry` and no circuits. The logic here checks for that and sets the variables appropriately.

Reply:
thats what i thought,  but the logic here, first sets the `circuits` to undefined,  then sets `registry=circuits` which is undefined,

Reply:
Ahh - of course. That makes sense.

Reply:
:+1:

Reply:
Need to switch the 2 lines in fact ?